bridge-impl/src/main/java/com/liferay/faces/bridge/application/internal/ResourceHandlerBridgeImpl.java

```diff
@@ -60,30 +57,6 @@ public ResourceHandlerBridgeImpl(ResourceHandler wrappedResourceHandler) {
 		this.wrappedResourceHandler = wrappedResourceHandler;
 	}
 
-	/**
-	 * Determines whether or not the specified URL has already been encoded.
-	 *
-	 * @param   url  The URL to check.
-	 *
-	 * @return  True if the specified URL is already encoded.
-	 */
-	public static boolean isEncodedFacesResourceURL(String url) {
```

Comment:
@ngriffin7a, `isEncodedFacesResourceURL()` and `isFacesResourceURL()` were unused, so I deleted them.